networking/ingress-node-firewall-operator.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="ingress-node-firewall-operator"]
+= Ingress Node Firewall Operator in {product-title}
+include::_attributes/common-attributes.adoc[]
+:context: ingress-node-firewall-operator
+
+toc::[]
+
+The Ingress Node Firewall Operator allows users to manage firewall configurations at the node level.
```

Comment:
But it's probably not _users_. It is likely that only a cluster administrator can modify node firewall rules.